nginx/config.go

```diff
@@ -145,6 +145,12 @@ http {
 			return 200;
 		}
 
+    {{ if $routerConfig.statusEndpoint }}
+    location /nginx_status {
+      stub_status on;
+    }
+    {{ end }}
+
```

Comment:
1, yes I'm aware, we're using mainly sysdig and the grafana part we skipped from our setup
2, sysdig wants me to have the nginx stub status exposed
3, sysdig cloud for the containers and newrelic for api performance metrics
4, 127.0.0.1:9090/nginx_status works just fine for me! :)